addons/barcodes_gs1_nomenclature/models/barcode_nomenclature.py

```diff
@@ -16,7 +16,7 @@ class BarcodeNomenclature(models.Model):
         string="Is GS1 Nomenclature",
         help="This Nomenclature use the GS1 specification, only GS1-128 encoding rules is accepted is this kind of nomenclature.")
     gs1_separator_fnc1 = fields.Char(
-        string="FNC1 Separator", trim=False, default=r'(Alt029|#|\x1D)',
+        string="FNC1 Separator", trim=False,
```

Comment:
We want to keep the `Alt029` part for the reason explained [here](https://github.com/odoo/odoo/commit/c2a41c5587effb9ad63580f6f2c50c7f97876d55).
I guess we can remove the `#` that said, if the user wants to use a custom separator, it has to configure its scan device anyway so it's not crazy to expect this field to be edited aswell.
```suggestion
        string="FNC1 Separator", trim=False, default=r'(Alt029|\x1D)',
```